docs/api-reference.mdx

```diff
@@ -60,6 +60,127 @@ const client = new MemoryClient({organizationId: "YOUR_ORG_ID", projectId: "YOUR
   </Tab>
 </Tabs>
 
+### Project Management Methods
+
+The Mem0 client provides comprehensive project management capabilities through the `client.project` interface:
+
+#### Get Project Details
+
+Retrieve information about the current project:
+
+```python
+# Get all project details
+project_info = client.project.get()
+
+# Get specific fields only
+project_info = client.project.get(fields=["name", "description", "custom_categories"])
+```
+
+#### Create a New Project
+
+Create a new project within your organization:
+
+```python
+# Create a project with name and description
+new_project = client.project.create(
+    name="My New Project",
+    description="A project for managing customer support memories"
+)
+```
+
+#### Update Project Settings
+
+Modify project configuration including custom instructions, categories, and graph settings:
+
+```python
+# Update project with custom categories
+client.project.update(
+    custom_categories=[
+        {"customer_preferences": "Customer likes, dislikes, and preferences"},
+        {"support_history": "Previous support interactions and resolutions"}
+    ]
+)
+
+# Update project with custom instructions
+client.project.update(
+    custom_instructions="Focus on customer satisfaction and personalized support"
+)
+
+# Enable graph memory for the project
+client.project.update(enable_graph=True)
```

Comment:
Are we listing somewhere that what all fields are possible to update in the `update()` method?